APP
------

try {
   app.init()
} catch(Exception ex) {
   log.error("Esto es un test")
   Sentry2.captureException(ex.message())
}
